src/metabase/models/interface.clj

```diff
@@ -157,9 +160,21 @@
       ;; the word "expression" but it is not MBQL (metabase#15882)
       (get viz-settings "graph.metrics")   (assoc :graph.metrics (get viz-settings "graph.metrics")))))
 
+;; TODO: probably not a perfect solution, but this lets corrupted card viz settings 'out', allowing cards to be recoverable
+(defn- maybe-normalize-visualization-settings
+  "Wraps viz-settings normalization and returns an empty map if normalization fails.
+  This may occur if the json string in the app-db is *technically* valid JSON, but doesn't normalize into a valid viz-settings map."
+  [viz-settings]
+  (try
+    (normalize-visualization-settings viz-settings)
+    (catch Throwable e
+      (log/error e (tru "Unable to normalize:") "\n"
+                 (u/pprint-to-str 'red viz-settings))
+      {})))
```

Comment:
If you pass 'nil' instead of an empty map, the app will load, but the app viz buttons don't cause changes. For example, I tried to change a bar chart's colors, but it would not change until I saved the question (thus saving valid viz-settings to the app-db), after which it worked fine.